CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes for each version of this project will be documented in this file.
 
+## 18.0.0
+### General
+- `IgxGrid`, `IgxTreeGrid`, `IgxHierarchicalGrid`, `IgxRowIsland`
+  - Removed deprecated property `displayDensity`.
```

Comment:
This will likely become a general bullet rather than just the grid, once the other components follow, but still:
```suggestion
  - Removed deprecated property `displayDensity`. Size is now controlled only though the custom CSS property `--ig-size`. Refer to the [Update Guide](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/update-guide) and components documentation for usage details.
```